src/link/MachO.zig

```diff
@@ -1746,15 +1766,21 @@ fn writeCodeSignaturePadding(self: *MachO) !void {
 }
 
 fn writeCodeSignature(self: *MachO) !void {
-    const code_sig_cmd = &self.load_commands.items[self.code_signature_cmd_index.?].LinkeditData;
+    const text_segment = self.load_commands.items[self.text_segment_cmd_index.?].Segment;
```

Comment:
Isn't it better to keep `code_signature_cmd_index` and update it when the patching is done and `writeCodeSignature` is called?